SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -30,4 +37,18 @@ class AppCenterViewController : NSViewController {
     UserDefaults.standard.set(userId, forKey: "userId")
```

Comment:
Could you please do some cleanup here and use `kMSUserIdKey`?

Reply:
Resolve in new commit.